app/models/commitment.rb

```diff
@@ -213,4 +215,8 @@ def self.pages(item_count)
   def user_created_and_live?
     live? && user_created?
   end
+
+  def name_is_10_words_or_less
+    errors.add(:name, :too_long) if name && name.split(' ').length > 10
```

Comment:
Would safe navigation or `try` work here, e.g. `name&.split(' ').length > 10`?

Reply:
we'd need two, but it would work. in this case i think it's slightly easier to read like this